src/platform_impl/macos/view.rs

```diff
@@ -131,6 +131,15 @@ fn retrieve_keycode(event: &NSEvent) -> Option<VirtualKeyCode> {
     })
 }
 
+fn mouse_button(event: &NSEvent) -> MouseButton {
+    match event.buttonNumber() {
+        0 => MouseButton::Left,
+        1 => MouseButton::Right,
+        2 => MouseButton::Middle,
```

Comment:
Is this constant documented anywhere?

Reply:
Not in the AppKit docs AFAIK. AppKit seems to completely ignore the relevance of the buttons _other_ than left/right. The closest thing in the Apple docs I was able to find is this constant in the CG/Quartz docs: https://developer.apple.com/documentation/coregraphics/cgmousebutton/kcgmousebuttoncenter. 
At least this is how both glfw and sdl handle it. And it works for my MX Vertical: middle = 2, back = 3, forward = 4.